fe1-web/src/features/evoting/components/ElectionVersionNotice.tsx

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StyleSheet, Text, View, ViewStyle } from 'react-native';
+
+import { PoPIcon } from 'core/components';
+import { Border, Color, Icon, Spacing, Typography } from 'core/styles';
+import STRINGS from 'resources/strings';
+
+import { Election, ElectionVersion } from '../objects';
+
+const styles = StyleSheet.create({
+  warning: {
+    padding: Spacing.x1,
+    marginBottom: Spacing.x1,
+    borderRadius: Border.radius,
+
+    flexDirection: 'row',
+    alignItems: 'center',
+  } as ViewStyle,
+  warningText: {
+    flex: 1,
+    marginLeft: Spacing.x1,
+  } as ViewStyle,
+  openBallot: {
+    backgroundColor: Color.warning,
+  } as ViewStyle,
+  secretBallot: {
+    backgroundColor: Color.success,
+  } as ViewStyle,
+});
+
+const ElectionVersionNotice = ({ election }: IPropTypes) => {
+  switch (election.version) {
+    case ElectionVersion.OPEN_BALLOT:
+      return (
+        <View style={[styles.warning, styles.openBallot]}>
+          <PoPIcon name="warning" color={Color.contrast} size={Icon.largeSize} />
+          <View style={styles.warningText}>
+            <Text style={[Typography.base, Typography.important, Typography.negative]}>
+              {STRINGS.general_notice}
+            </Text>
+            <Text style={[Typography.base, Typography.negative]}>
+              {STRINGS.election_warning_open_ballot}
```

Comment:
I would also put this as a typical blue-ish info box, say, something in this style.
https://www.tutorialrepublic.com/lib/images/bootstrap-5/bootstrap-alert-messages-with-icons.png

Reply:
That was one of the things I wanted to discuss! Yes, I think having a blue-ish info box is better!